superbench/benchmarks/micro_benchmarks/cuda_common.cmake

```diff
@@ -1,10 +1,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
-if(NOT DEFINED CMAKE_CUDA_STANDARD)
-    set(CMAKE_CUDA_STANDARD 11)
-    set(CMAKE_CUDA_STANDARD_REQUIRED ON)
-endif()
+enable_language(CUDA)
```

Comment:
please do not enable_language(CUDA) in cuda_common.cmake, it will raise error and failed to make anyone in rocm environment

Reply:
Done.